README.adoc

```diff
@@ -1,8 +1,32 @@
-== Spring Cloud Scheduler image:https://build.spring.io/plugins/servlet/wittified/build-status/SCD-SCHED[Build Status, link=https://build.spring.io/browse/SCD-SCHED] image:https://badge.waffle.io/spring-cloud/spring-cloud-scheduler.svg?label=ready&title=Ready[Stories Ready, link=http://waffle.io/spring-cloud/spring-cloud-scheduler] image:https://badge.waffle.io/spring-cloud/spring-cloud-scheduler.svg?label=In%20Progress&title=In%20Progress[Stories In Progress, link=http://waffle.io/spring-cloud/spring-cloud-scheduler]
+//:image-root: https://raw.githubusercontent.com/spring-cloud/spring-cloud-scheduler/master/images
+
+= Spring Cloud Scheduler
 
 The Spring Cloud Scheduler project defines an SPI for scheduling short lived tasks.
 
-=== Building
+== Overview
+This project offers an interface that abstracts the common functionality
+for scheduling ephemeral applications on cloud platforms using their native scheduler (Pivotal Cloud Foundry Scheduler, Kubernetes Cron Job).
```

Comment:
Perhaps:

> platforms using their native scheduler (Pivotal Cloud Foundry Scheduler, Kubernetes Cron Job).

to:

> platforms using their native scheduler such as Pivotal Cloud Foundry Scheduler and Kubernetes Cron Job to schedule applications in Pivotal Cloud Foundry and Kubernetes respectively.

Reply:
If you go with Sabby's suggetion (works for me, because it adds detail), I'd write it as follows:

platforms by using their native schedulers, such as Pivotal Cloud Foundry Scheduler and Kubernetes Cron Job, to schedule applications in Pivotal Cloud Foundry and Kubernetes, respectively.